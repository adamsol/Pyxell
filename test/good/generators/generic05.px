
func inverse(a) def
    yield a
    if a == 0 do
        return
    yield 1/a

print [...inverse(5)]
print [...inverse(0f)]
