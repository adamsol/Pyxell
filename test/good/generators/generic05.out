[5, 0.2]
[0]
